package.json

```diff
@@ -64,14 +64,16 @@
         "spectrum-tokens": "node ./scripts/spectrum-tokens.js",
         "spectrum-vars": "node ./scripts/spectrum-vars.js",
         "start": "yarn storybook",
-        "storybook": "yarn build:ts && yarn build:css && run-p storybook:start watch:css build:watch",
+        "storybook": "yarn build && yarn storybook:quick",
         "storybook:build": "build-storybook -o projects/documentation/dist/storybook",
-        "storybook:start": "web-dev-server --config wds-storybook.config.js",
+        "storybook:quick": "run-p watch:css build:watch storybook:run",
+        "storybook:run": "wds --config wds-storybook.config.js",
```

Comment:
I saw some great advice the other day that recommended not using shorthand in npm scripts since they're more self-documenting when fully spelled out. Might be nice to do that here for `wds` and use `web-dev-server` instead so it's really clear what package is being called.

Reply:
If this is useful to people, sounds good! Updating.